tests/functional/prometheous/test_prometheus_validations.py

```diff
@@ -974,3 +975,82 @@ def test_restart_prometheus_glusterfs_pod(self):
             # Try to fetch metric from prometheus pod
             self._fetch_metric_from_promtheus_pod(
                 metric='heketi_device_brick_count')
+
+    @pytest.mark.tier2
+    def test_metrics_workload_on_prometheus(self):
+        """Validate run metrics workload on prometheus"""
+
+        self.prefix = "autotest-{}".format(utils.get_random_str())
+
+        # Add check for CRS version
+        openshift_ops.switch_oc_project(
+            self._master, self._registry_project_name)
+        if not self.is_containerized_gluster():
+            self.skipTest("Skipping this test case as CRS"
+                          " version check can not be implemented")
+
+        # Get one of the prometheus pod name and respective pvc name
+        openshift_ops.switch_oc_project(
+            self._master, self._prometheus_project_name)
+        prometheus_pods = openshift_ops.oc_get_pods(
+            self._master, selector=self._prometheus_resources_selector)
+        if not prometheus_pods:
+            self.skipTest(
+                prometheus_pods, "Skipping test as prometheus"
+                " pod is not present")
+
+        prometheus_pod = list(prometheus_pods.keys())[0]
+        pvc_name = openshift_ops.oc_get_custom_resource(
+            self._master, "pod",
+            ":.spec.volumes[*].persistentVolumeClaim.claimName",
+            prometheus_pod)
+        self.assertTrue(
+            pvc_name, "Failed to get PVC name for prometheus"
+            " pod {}".format(prometheus_pod))
+        pvc_name = pvc_name[0]
+        self.verify_iscsi_sessions_and_multipath(
+            pvc_name, prometheus_pod, rtype='pod',
+            heketi_server_url=self._registry_heketi_server_url,
+            is_registry_gluster=True)
```

Comment:
```suggestion
        pvc_custom = ":.spec.volumes[*].persistentVolumeClaim.claimName"
        pvc_name = openshift_ops.oc_get_custom_resource(
            self._master, "pod", pvc_custom, prometheus_pod)
        self.assertTrue(
            pvc_name, "Failed to get PVC name for prometheus"
            " pod {}".format(prometheus_pod))
        self.verify_iscsi_sessions_and_multipath(
            pvc_name[0], prometheus_pod, rtype='pod',
            heketi_server_url=self._registry_heketi_server_url,
            is_registry_gluster=True)
```

Reply:
@adityaramteke ; I do not understand the purpose of adding 'pvc_custom' as a variable as we are not using it anywhere else?

Reply:
@sayaleeraut, we are defining the custom variable just for the simplicity of the code. Please refer to one of the TC https://github.com/gluster/glusterfs-containers-tests/blob/master/tests/functional/logging/test_logging_validations.py#L115

Reply:
@adityaramteke; okay, I will add it.